internal/builtin/providers/tf/data_source_state.go

```diff
@@ -114,7 +115,9 @@ func dataSourceRemoteStateRead(d cty.Value, enc encryption.StateEncryption, path
 		return cty.NilVal, diags
 	}
 
-	configureDiags := b.Configure(cfg)
+	// TODO: Plumb a real context in here, once [providers.Interface] has
```

Comment:
A TODO for later or was it meant to be fixed in this PR?

Reply:
From my understanding, this is a real TODO.